src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -18,13 +18,17 @@
     PythonModuleOwners,
     PythonModuleOwnersRequest,
     ResolveName,
+    map_module_to_address,
 )
 from pants.backend.python.dependency_inference.parse_python_dependencies import (
     ParsedPythonAssetPaths,
     ParsedPythonDependencies,
     ParsedPythonImports,
     ParsePythonDependenciesRequest,
 )
+from pants.backend.python.dependency_inference.parse_python_dependencies import (
+    parse_python_dependencies as parse_python_dependencies_get,
```

Comment:
Frustratingly, we have a module called parse_python_dependencies with a function called parse_python_dependencies - while also needing to sometimes refer to the parse_python_dependencies.rules() - so something was getting aliased for sure.